report/renderer/crc-details-renderer.js

```diff
@@ -14,46 +14,26 @@ import {Globals} from './report-globals.js';
 /** @typedef {import('./dom.js').DOM} DOM */
 /** @typedef {import('./details-renderer.js').DetailsRenderer} DetailsRenderer */
 /**
- * @typedef CRCSegment
- * @property {LH.Audit.Details.SimpleCriticalRequestNode[string]} node
+ * @typedef NetworkSegment
+ * @property {LH.Audit.Details.SimpleCriticalRequestNode[string]|LH.Audit.Details.NetworkNode[string]} node
  * @property {boolean} isLastChild
  * @property {boolean} hasChildren
- * @property {number} startTime
- * @property {number} transferSize
```

Comment:
`startTime` and `transferSize` are never actually used anywhere. Instead, this file will just look at the data in `node`.